controllers/projects_controller.go

```diff
@@ -26,12 +26,19 @@ type EditProjectForm struct {
 	Description string `param:"description"`
 }
 
+type SettingsProjectForm struct {
```

Comment:
__[golint]__
* [golint] exported type SettingsProjectForm should have comment or be unexported